src/solver/variable/economy/thermalAirPollutantEmissions.h

```diff
@@ -24,25 +24,20 @@
 **
 ** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
 */
-#ifndef __SOLVER_VARIABLE_ECONOMY_CO2_H__
-#define __SOLVER_VARIABLE_ECONOMY_CO2_H__
+#ifndef __SOLVER_VARIABLE_ECONOMY_thermalAirPollutantEmissions_H__
+#define __SOLVER_VARIABLE_ECONOMY_thermalAirPollutantEmissions_H__
 
+#include <antares/study.h>
 #include "../variable.h"
 
-namespace Antares
+namespace Antares::Solver::Variable::Economy
 {
-namespace Solver
+struct VCardThermalAirPollutantEmissions
 {
-namespace Variable
-{
-namespace Economy
-{
-struct VCardCO2
-{
-    //! Caption
+    //! Caption not used: several columns
     static const char* Caption()
     {
-        return "CO2 EMIS.";
+        return "";
```

Comment:
This is used internally, please fill a value 
```suggestion
        return "POLLUTANT EMISS.";
```